sigal/themes/galleria/templates/index.html

```diff
@@ -154,6 +130,24 @@ <h2>
     <script src="{{ theme.url }}/themes/classic/galleria.classic.min.js"></script>
     <script src="{{ theme.url }}/plugins/history/galleria.history.min.js"></script>
     <script>
+var data = [
+  {% for media in album.medias -%}
+  {
+    title: "{{ media.title if media.title else media.filename }}",
+    description: "{{ img_description(media) | e }}",
+    {% if media.type == "image" %}
+      image: "{{ media.filename }}",
+      thumb: "{{ media.thumbnail }}"
+      {%- if media.big %},
+      big: "{{ media.big }}"{% endif %}
+    {% endif %}
+    {% if media.type == "video" %}
+      video: "{{ media.filename }}"
```

Comment:
So it seems that thanks to this the /img/empty.png file is not more necessary ? Do you know how galleria proceed for the thumbnail ?

Reply:
I don’t have videos in my regular galleries, but when I did add one a thumbnail was generated, so I’ve made an update to use that thumbnail, and removed `/img/empty.png`